test-cloud/pom.xml

```diff
@@ -128,6 +128,16 @@
                 </files>
               </configuration>
             </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-failsafe-plugin</artifactId>
+              <configuration>
+                <excludes>
+                  <!-- Test use template which is available just for PostgreSQL -->
+                  <exclude>**/EmptyManagedKieServerConnectionIntegrationTest.java</exclude>
```

Comment:
Maybe, we can consider to use junit categories for exclusions. I think that it will be quite painful to exclude test classes one by one.

Reply:
Well, categories aren't as good solution either as they are specific for a test.
Maybe the way would be to have a check on scenario level, verifying that the scenario is available. @jiripetrlik What do you think?

Reply:
What do you mean by "scenario level". How can it be implemented?

Reply:
For example by adding method DeploymentScenario.isAvailable() . Will return true is the scenario has all its resources ready to be used.
Each test could have then assumeTrue(deploymentScenario.isAvailable()) .

Reply:
Well, it sounds like a good solution.